journal/modernized/06-chapter-6.adoc

```diff
@@ -252,17 +258,26 @@ His body was the next day attended by a large number of people
 of various religious denominations to our meeting house,
 and afterwards interred in Friends burial ground in this city.
 
+[.signed-section-context-close]
 Signed on behalf, and by appointment of our monthly meeting, held in Philadelphia,
 by adjournment, the 2nd of the eighth month 1770.
 
+[.signed-section-signature]
 James Pemberton, Clerk.
 
-* * *
+--
+
+[.asterism]
```

Comment:
There was a long line here separating these testimionies originally, and I thought I remember replacing those lines with asterisks, but I'm not totally sure!

Reply:
Yup, that's correct 👍🏻